project/plugins.sbt

```diff
@@ -8,4 +8,6 @@ addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.2.7")
 
 addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.13")
 
-addSbtPlugin("com.evolutiongaming" % "sbt-scalac-opts-plugin" % "0.0.5")
\ No newline at end of file
+addSbtPlugin("com.evolutiongaming" % "sbt-scalac-opts-plugin" % "0.0.5")
+
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.5.1")
```

Comment:
why do we need this here? @saraiva132 you could just add that in your local `.sbt/` folder

Reply:
I removed it ;)

Reply:
it is indeed super useful! hence just add it locally to have for all projects, the idea is that you don't add optional stuff into project files, but rather into your local config :)